topostats/io.py

```diff
@@ -86,6 +103,32 @@ def write_yaml(
             LOGGER.error(exception)
 
 
+def write_config_with_comments(config: str, filename: str = "config.yaml"):
```

Comment:
Minor pedantic typehint for the return value.

```suggestion
def write_config_with_comments(config: str, filename: str = "config.yaml") -> None:
```